guidedmodules/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 from django import forms
 from django.utils.html import escape as escape_html
+from django.contrib import messages
```

Comment:
This line doesn't seem to be needed. Or if it fixes a different issue, it's not mentioned in the commit message.

Reply:
Line removed.